File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/c/cwe-476/code_before2.c
// From cwe-snippets, snippets_90/non-compliant/C/cwe-0476/long_04.c

void func()
{
    long * data;
    if(STATIC_CONST_TRUE)
    {
        data = NULL;
    }
    if(STATIC_CONST_TRUE)
    {
        printLongLine(*data);
    }
}

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/java/non-vul/code_before2.java
// From cwe-snippets, snippets_90/non-compliant/Java/cwe-0114/basic_10.java

private void func() throws Throwable
{
    if (IO.staticTrue)
    {
        String root;
        String libraryName = "test.dll";
        if(System.getProperty("os.name").toLowerCase().indexOf("win") >= 0)
        {
            /* running on Windows */
            root = "C:\\libs\\";
        }
        else
        {
            /* running on non-Windows */
            root = "/home/user/libs/";
        }
        System.load(root + libraryName);
    }
}
